src/Bart/Jenkins/Connection.php

```diff
@@ -0,0 +1,107 @@
+<?php
+namespace Bart\Jenkins;
+use Bart\Diesel;
+use Bart\JSON;
+use Bart\Log4PHP;
+use Bart\Primitives\Strings;
+
+/**
+ * Class Connection
+ * @package Bart\Jenkins
+ */
+class Connection
```

Comment:
Great looking class. A brief line at the top would be nice, `Manages interface between remote Jenkins REST API and client code`, for example.